language/stdlib/src/transaction_scripts.rs

```diff
@@ -37,6 +37,7 @@ pub enum StdlibScript {
     CreateAccount,
     CreateEmptyAccount,
     EmptyScript,
+    Freeze,
```

Comment:
FreezeAccount?

Reply:
Not entirely sure why `stdlib.mv` would need to be updated with this. That should only change if there are changes to the stdlib modules unless I'm mistaken (and I don't see any changes to them).